modified the sentence in a more understandable way (#11140)
grafana/loki: README.md

Loki can be run in a single host, no-dependencies mode using the following commands.

> You need to install `go`, and make sure to use the `go` version found in [our build Dockerfile](https://github.com/grafana/loki/blob/main/loki-build-image/Dockerfile) otherwise it may give you an error.

Reviewer: ```suggestion
You need to install `go`.  We recommend using the `go` version found in [our build Dockerfile](https://github.com/grafana/loki/blob/main/loki-build-image/Dockerfile) to avoid errors.
```

Author: @JStickler Do you want me to edit this message with a new one that you recommended?

Reviewer: Yes please.
